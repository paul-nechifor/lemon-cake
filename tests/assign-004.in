(= a 2)
(= b (+ a 1))
(= c (+ b 100))
c
b
a
