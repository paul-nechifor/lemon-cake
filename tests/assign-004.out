2
3
103
103
3
2
